ui/imports/utils/Utils.qml

```diff
@@ -871,7 +871,7 @@ QtObject {
     }
 
     function isBase64DataUrl(str) {
-        return /^data:.*;base64,/.test(str);
+        return globalUtilsInst.isBase64DataUrl(str)
```

Comment:
The general direction of many refactors done so far is, beside others, to remove direct access to nim's context properties in global singletons. Those singletons should be backend-independent and stateless. It helps a lot regarding modularization. Any UI component depending on such method won't be fully independent UI component.

The recommended solution is to expose such methods via not-singleton store.

Reply:
Igor dodging responsibility like Neo dodging bullets :joy: